packages/core-components/package.json

```diff
@@ -34,6 +34,7 @@
   },
   "dependencies": {
     "@backstage/config": "^1.0.1",
+    "@backstage/core-app-api": "^1.0.4-next.0",
```

Comment:
Hmm...  I'm not 100% sure we want to take this dependency in core components.  Might be safer to just duplicate the logic of `getBasePath` privately in the link component.